tests/phpunit/tests/post/primePostParentIdCaches.php

```diff
@@ -94,17 +94,17 @@ public function test_prime_post_parents_caches_update() {
 		);
 
 		$before_num_queries = get_num_queries();
-		_prime_post_parents_caches( array( $page_id ) );
+		_prime_post_parent_id_caches( array( $page_id ) );
 		$num_queries = get_num_queries() - $before_num_queries;
 
-		$this->assertSame( 1, $num_queries, 'Unexpected number of queries on second run' );
+		$this->assertSame( 0, $num_queries, 'Unexpected number of queries on second run' );
```

Comment:
```suggestion
		$this->assertSame( 1, $num_queries, 'Unexpected number of queries on second run' );
```

Reply:
Missed that this change had snuck in with the name change. 👍🏻